docusaurus/docs/core-concepts/security/sessions.md

```diff
@@ -1,9 +1,7 @@
-# API Session
+## API Session {#api-session}
 
 API Sessions represent a client that is either partially or fully authenticated as a specific Ziti Identity. API
-Sessions are used to scope [authentication](./authentication/auth) and [Posture Data](./authorization/posture-checks).
```

Comment:
why are you removing these lines? this is not relevant to links being broken?

Reply:
I'm sure that's my mistake. The link was broken, but it should be fixed, not removed. Will patch.